packages/newspringchurchapp/src/prayer/PrayerSingle/PrayerSingle.js

```diff
@@ -0,0 +1,100 @@
+import React, { PureComponent } from 'react';
+import { View } from 'react-native';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import {
+  BodyText,
+  H5,
+  styled,
+  Touchable,
+  ChannelLabel,
+  SideBySideView,
+} from '@apollosproject/ui-kit';
+import PrayerHeader from '../PrayerHeader';
+
+const GreyH5 = styled(({ theme }) => ({
+  color: theme.colors.text.tertiary,
+  marginBottom: theme.sizing.baseUnit * 0.5,
+}))(H5);
+
+const PrayerText = styled(({ theme }) => ({
+  marginVertical: theme.sizing.baseUnit,
+}))(BodyText);
+
+class PrayerSingle extends PureComponent {
+  static navigationOptions = () => ({
+    header: null,
+  });
+
+  render() {
+    const {
+      showHelp,
+      header,
+      avatarSize,
+      prayer,
+      action,
+      navigation,
+    } = this.props;
+
+    return (
+      <View>
+        <GreyH5>
+          {prayer.enteredDateTime
```

Comment:
I think I'd like to see this removable via prop. Mostly because if we're sticking to the design we were given, this doesn't exist on the cards on the prayer list, but it _does_ exist on the users prayer list cards.

Reply:
agreed